operations/boot_orchestration/Multi_tenancy_with_BOS.md

```diff
@@ -21,15 +21,38 @@ resources specific to that tenant.
 
 ### Components
 
-Component ownership is not directly tracked by BOS to avoid duplicating information stored in TAPMS and HSM. As a result
+Component ownership is not directly tracked by `BOS` to avoid duplicating information stored in `TAPMS` and HSM. As a result
 the system administrator will not see any tenant information in the component records and the components endpoint will function
-normally for system administrators. To determine which components belong to which tenant, the administrator will need to query TAPMS.
+normally for system administrators. To determine which components belong to which tenant, the administrator will need to query `TAPMS`.
 
 ## Sessions and session templates
 
 Both session and session template records now contain information on the tenant that owns them. System administrators can list
-all sessions and session templates across all tenants. However system administrators will not be able to create, retrieve,
+all sessions and session templates across all tenants. However, system administrators will not be able to create, retrieve,
 update, or delete specific records that belong to a tenant without authenticating to that specific tenant. This is
 because the resources are name-spaced to avoid naming collisions between tenants. The name-spacing means that there can
-be two or more records with the same name, and BOS does not know which resource is requested unless the tenant is
+be two or more records with the same name, and `BOS` does not know which resource is requested unless the tenant is
 specified. To access tenant owned resources, the system administrator will need to masquerade as that tenant.
+
+## BOS API Access with Tenancy
+
+`BOS` v2 API introspects request information pertaining to tenant information, if provided. If you have tenant information
+populated by the CrayCLI, it will be passed into the associated `BOS` command. Otherwise, if you are interacting with the
+API directly and are scripting against it, you may provide this information by hand, via curl.
+
+Note: `OPA` prevents unauthorized tenant requests from accessing the `BOS` API, so the value you choose for 
+`Cray-Tenant-Name: ` must match your user provided access token within your request. If your user access token is not
```

Comment:
This user-provide access token is the bearer token indicated in the example as
Authorization: Bearer ${TOKEN}
correct?

Reply:
Yes.